scania-digital-design-system/sdds
# refactor(storybook):increased testability

**Describe pull-request**  
- Set 'padded' as global default layout for stories so it's easier to manually check visual appearance
- Set 'centered' as layout for components needing this (tooltip, spinner)
- Remove excess demo code, such as margins and padding, to avoid false positive or negative visual test results
- Slight refactor on tooltip stories to increase testability
- Adding more Lorem ipsum to accordion to better visualise what the paddingReset-control does
- Components that doesn't have a default width has been set to 400 for demo purposes (card, slider, textarea, accordion)
- Change control types from select to radio to make control values more clear

Bonus:
- Moving fullpage-pattern story from /theme to /patterns so it's easier to find. (When refactoring header and side menu, we've forgot to update these stories which made them break)

**Solving issue**  
_In case of GitHub issue add # plus the number of the issue (for example #123) OR if it is Azure then AB# and number of ticket_
Fixes: [AB#1678](https://dev.azure.com/scaniadigitaldesignsystem/3d9eb9c6-0045-473d-bc1d-e842dd779200/_workitems/edit/1678) [AB#1693](https://dev.azure.com/scaniadigitaldesignsystem/3d9eb9c6-0045-473d-bc1d-e842dd779200/_workitems/edit/1693)

**How to test**  
_Add description how to test if possible_
1. Go to storybook preview
2. Look through all components
3. Notice if layout should be different on any of the components
4. Notice if control types are either text, toggle or radio

**Screenshots**  
_If applicable, add screenshots to help explain_

![image](https://user-images.githubusercontent.com/97448894/154661058-622a0848-058f-44c9-855c-ea811eb3a2e9.png)
